webapp/src/containers/Rhs/index.tsx

```diff
@@ -256,14 +256,14 @@ const Rhs = (): JSX.Element => {
                 </div>
             )}
             <Dialog
-                description={dialogContent}
-                destructive={showDestructivetDialog}
+                destructive={showDestructiveDialog}
                 show={showDisconnectDialog}
-                primaryActionText={showDestructivetDialog && 'Disconnect'}
+                primaryButtonText={'Disconnect'}
```

Comment:
same

Reply:
`showDestructiveDialog` is always true when the `showDisconnectDialog` is true vis-a-vis `primaryActionText` will always show up as Disconnect when modal is visible. The removed condition didn't add any value

Reply:
reverted as discussed